### PR TITLE
Change tags ordering from Z-A to A-Z

### PR DESCRIPTION
**Pull request by Dimitar5555**

### CODE REVIEW

File: osmorg-editor.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ function queryForTags() {
                 tags.unshift(k + ' = ' + v);
             }
         }
-        textArea.value = tags.join('\n');;
+        textArea.value = tags.reverse().join('\n');;
```
Comment (Zverik) [inline osmorg-editor.js]: Wouldn't it be more efficient to replace `tags.unshift` above with a proper function?

Comment (Dimitar5555) [reply osmorg-editor.js]: I though that `tags.unshift` did something else. We can just use `tags.push` instead and there won't be a need to use `.reverse()`.

**Comment by Zverik** [comment]
Thanks!

**Comment by Dimitar5555** [comment]
@Zverik Is this change released on Firefox's addons site? The latest update there is from July 2022.

**Pull request merged**